lib/jsdom/living/attributes.js

```diff
@@ -28,21 +30,40 @@ exports.changeAttribute = function (element, attribute, value) {
   // https://dom.spec.whatwg.org/#concept-element-attributes-change
 
   const { _localName, _namespace, _value } = attribute;
+
   queueAttributeMutationRecord(element, _localName, _namespace, _value);
 
-  const oldValue = attribute._value;
+  if (element._ceState === CUSTOM_ELEMENT_STATE.CUSTOM) {
```

Comment:
I'd rather just use strings directly like the spec does, instead of strings after a `.`